### PR TITLE
[Middleware_02] Build MiddlewareCommInterface

### PR DESCRIPTION
**Pull request by NMoua-Edu**
working on #70

### CODE REVIEW

**Comment by NMoua-Edu** [comment]
got a MiddlewareCommInterfaceImplementation Unit Tests > Rfid In Is the same Rfid Out FAILED
    java.lang.IllegalArgumentException at MiddlewareCommInterfaceImplementationTests.java:22

File: src/main/java/edu/oakland/production/middleware02/MiddlewareCommInterfaceImplementation.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    if(middlewareManager == null) {
+    throw new IllegalArgumentException("middlewareManager cannot be null");
+    }
+    this.middlewareManager = middlewareManager;
+  }
+
+  /**
+   * Receives the rfid for Use Case 1.
+   * 
+   * @return rfid.
```
[the remark this answers is not in the document]
Comment (andrewdimmer) [reply src/main/java/edu/oakland/production/middleware02/MiddlewareCommInterfaceImplementation.java:32]: You need to remove the space after the `*` on line 29... took us almost an hour to debut that with another team!

File: src/main/java/edu/oakland/production/middleware02/MiddlewareCommInterfaceImplementation.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+*
+*/
+public class MiddlewareCommInterfaceImplementation implements MiddlewareCommInterface {
+  int rfid;
+  
+  private MiddlewareCommLinkManager middlewareManager;
+  /** 
+   * Creates a MiddlewareCommInterfaceImplementation to recive the Rfid.
+   * 
+   * @param middlewareManager the class that this class will pass the Rfid to.
```
[the remark this answers is not in the document]
Comment (andrewdimmer) [reply src/main/java/edu/oakland/production/middleware02/MiddlewareCommInterfaceImplementation.java:18]: You need to remove the space after the `*` on line 15... took us almost an hour to debut that with another team!

**Review by andrewdimmer — COMMENTED** [review]
Let me know once finish the last of the changes for the style guide, and then you should be good to merge!

**Review by mckenzieking — APPROVED** [review]
glad this is fixed

**Review by kwillard34 — APPROVED** [review]

**Pull request merged**